web_src/js/index.js

```diff
@@ -3556,6 +3556,20 @@ window.onOAuthLoginClick = function () {
   }, 5000);
 };
 
+window.onGitIgnorePopulate = function (element) {
```

Comment:
I'm not entirely sure if this is the correct place to put it. Is there somewhere else to include page-specific functionality?

Reply:
I suggest [here](https://github.com/go-gitea/gitea/blob/6a62884782b29653bcd4462188e263018e5e21e8/web_src/js/index.js#L756)